src/Modules/Settings/Dnn.PersonaBar.SiteSettings/Services/SiteSettingsController.cs

```diff
@@ -3176,12 +3172,21 @@ private string DisplayDataType(int dataType)
             return retValue;
         }
 
-        private bool ValidateProperty(ProfilePropertyDefinition definition)
+        private bool ValidateProperty(ProfilePropertyDefinition definition, out HttpResponseMessage httpPropertyValidationError)
         {
             bool isValid = true;
+            httpPropertyValidationError = null;
             var objListController = new ListController();
             string strDataType = objListController.GetListEntryInfo("DataType", definition.DataType).Value;
 
+            Regex propertyNameRegex = new Regex("^[a-zA-Z0-9]+$");
```

Comment:
You updated the other regex but not here. Should both be the same?

Reply:
In Javascript, the regex /^[a-zA-Z0-9]+$/ return true if the variable is null or undefined. This is not the case in C#

Reply:
This is not what I was asking. In JS you the expression now starts with `\w*` but not in C#

Reply:
My mistake. What I really wanted is + instead of \w*. Corrected now.

Reply:
Really, so we want to accept name like `A++B` or `++XYZ`? 🤕